internal/build/build.go

```diff
@@ -16,4 +17,7 @@ func init() {
 			Version = info.Main.Version
 		}
 	}
+
+	// https://github.com/gdamore/tcell/commit/2f889d79bd61b1fd2f43372529975a65b792a7ae
+	_ = os.Setenv("TCELL_MINIMIZE", "1")
```

Comment:
Non blocking but if I saw this I'd really like to see a bit more commentary as to why we're doing this here without having to navigate through to the PR.

Reply:
What are the consequences are of setting this for commands that do use `tcell`?

Reply:
Good questions. The consequence is that some Unicode glyphs might cause mis-alignment when rendered inside tcell views. I'll add this to the comment.

Reply:
> The consequence is that some Unicode glyphs might cause mis-alignment when rendered inside tcell views

my reading of the code is that we're just losing some precaching in tcell. I tested ext browse and didn't see any issues around unicode characters, though it was not an exhaustive test.